set.mm

```diff
@@ -748683,6 +748793,185 @@ set of all the singletons of (proper) ordered pairs over the elements of
+      ( wfo ccnv cdm cima cres ffnd eqid rescnvimafod fcoreslem1 reseq2d syl5eq
+      eqidd foeq123d mpbird ) ADEHMFNGOZPZEFUHQZMABUGUHEFABCFIRJUHSTADUHEEHUIAH
+      FDQUILADUHFABCDEFGIJKUAZUBUCUJAEUDUEUF $.
+
+    fcores.g $e |- ( ph -> G : C --> D ) $.
+    fcores.x $e |- X = ( G |` E ) $.
+    $( Lemma 4 for ~ fcores .  (Contributed by AV, 17-Sep-2024.) $)
+    fcoreslem4 $p |- ( ph -> ( X o. Y ) Fn ( `' F " C ) ) $=
+      ( wfn cima crn ccnv wss ccom cres ffnd cdm cin inss2 a1i wceq fdmd eqcomd
+      3sstr4d fneq1i sylibr wfo wf fcoreslem3 fof syl imaeq2d fcoreslem1 eqtr4d
+      fnssresd fneq2d mpbird fcoreslem2 eqimss fnco syl3anc ) AJGRZKHUAZDSZRZKT
+      ZGUBZJKUCVMRAIGUDZGRVKADGIADEIPUEAHTZIUFZUGZVSGDVTVSUBAVRVSUHUIGVTUJAMUIA
+      VSDADEIPUKULZUMVDGJVQQUNUOAVNKFRAFGKAFGKUPFGKUQABCFGHIKLMNOURFGKUSUTUEAVM
+      FKAVMVLVSSFADVSVLWAVAABCFGHILMNVBVCVEVFAVOGUJVPABCFGHIKLMNOVGVOGVHUTGVMJK
+      VIVJ $.
+
+    $d C x $.  $d F x $.  $d G x $.  $d X x $.  $d Y x $.  $d ph x $.
+    $( Every composite function ` ( G o. F ) ` can be written as composition of
+       restrictions of the composed functions (to their minimum domains).
+       (Contributed by GL and AV, 17-Sep-2024.) $)
+    fcores $p |- ( ph -> ( G o. F ) = ( X o. Y ) ) $=
+      ( wcel cfv eleq2d vx ccnv cima ccom wf wfun ffund fcof syl2anc fcoreslem4
+      ffnd cv wa cres fveq1i fcoreslem1 fdmd imaeq2d eqtrd eqcomd biimpa fvresd
+      cdm syl5eq fveq2d crn cin cnvimass sseli fvelrn syl2an fvimacnvi sylan wb
+      adantr mpbird eleqtrrdi wfo fcoreslem3 fof syl fvco3d wss sselda 3eqtr4rd
+      elind a1i eqfnfvd ) AUAHUBZDUCZIHUDZJKUDZAWJEWKADEIUEHUFZWJEWKUEPABCHLUGZ
+      DEIHUHUIUKABCDEFGHIJKLMNOPQUJAUAULZWJRZUMZWOKSZJSZWOHSZISZWOWLSWOWKSWQWSW
+      TJSZXAWQWRWTJWQWRWOHFUNZSWTWOKXCOUOWQWOFHAWPWOFRZAWJFWOAFWJAFWIIVCZUCZWJA
+      BCFGHILMNUPZAXEDWIADEIPUQZURUSUTTVAVBVDVEWQXBWTIGUNZSXAWTJXIQUOWQWTGIWQWT
+      HVFZXEVGGWQXJXEWTAWMWOHVCZRZWTXJRWPWNWJXKWOHDVHZVIWOHVJVKWQWTXERZWTDRZAWM
+      WPXOWNWODHVLVMAXNXOVNWPAXEDWTXHTVOVPWFMVQVBVDUSWQFGWOJKAFGKUEZWPAFGKVRXPA
+      BCFGHIKLMNOVSFGKVTWAVOWQXDWOXFRZAWPXQAWJXFWOADXEWIAXEDXHUTURTVAAXDXQVNWPA
+      FXFWOXGTVOVPWBWQBCWOIHABCHUEWPLVOWQWOBRZXLAWJXKWOWJXKWCAXMWGWDAXRXLVNWPAB
+      XKWOAXKBABCHLUQUTTVOVPWBWEWH $.
+
+    $( Lemma for ~ fcoresf1 .  (Contributed by AV, 18-Sep-2024.) $)
+    fcoresf1lem $p |- ( ( ph /\ Z e. P )
+                        -> ( ( G o. F ) ` Z ) = ( X ` ( Y ` Z ) ) ) $=
+      ( ccom cfv wcel wa wceq fcores adantr fveq1d wfo fcoreslem3 fof syl simpr
+      wf fvco3d eqtrd ) ALFUAZUBZLIHSZTLJKSZTLKTJTUPLUQURAUQURUCUOABCDEFGHIJKMN
+      OPQRUDUEUFUPFGLJKUPFGKUGZFGKULAUSUOABCFGHIKMNOPUHUEFGKUIUJAUOUKUMUN $.
+
+    ${
+      $d E x y $.  $d F y $.  $d G y $.  $d P a b x y $.  $d X a b y $.
+      $d Y a b y $.  $d ph a b y $.
+      fcoresf1.i $e |- ( ph -> ( G o. F ) : P -1-1-> D ) $.
+      $( If a composition is injective, then the minimum restrictions of its
```

Comment:
"minimum restriction" --> "restriction to the smallest..."

(Because to "minimally restrict" is to not restrict at all.)